gulpfile.js

```diff
@@ -35,7 +35,8 @@ gulp.task('watch', function() {
 
 gulp.task('connect', function() {
   connect.server({
-    root: 'app/'
+    root: 'app/',
+    port: 3000 // Had an issue with default of 8080
```

Comment:
Feel free to ignore this one, or implement it.  I had an issue with 8080 as my Mac already has a localhost that broadcasts on this port and I was getting conflicts